otj-server-core/src/test/java/com/opentable/server/PortSelectionWithNamedPortsAndAClashInKubernetesTest.java

```diff
@@ -44,7 +44,7 @@
         "ot.jmx.port=5561",
         "TASK_HOST=mesos-slave9001-dev-sf.qasql.opentable.com",
         "IS_KUBERNETES=TRUE",
-        "ot.httpserver.active-connectors=default-http,my-https,boot"
+        "ot.httpserver.active-connectors=default-http,my-https"
```

Comment:
Now we cannot have both `default-http` and `boot` simultaneously. It actually failing with `Port 5559 already in use`.

Reply:
do we test for where default-http is replaced with boot? or that does not make sense.